lib/web_ui/test/canvaskit/fallback_fonts_test.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Tests that run `matchGoldenFile` should have `_golden_test.dart` in their names. Otherwise, our test runner will attempt to run the test in parallel and take random screenshots.

Reply:
Done.